Assets/Scripts/Editor/BuildScript.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+namespace Editor
+{
+    public class BuildScript
+    {
+        static void Windows()
+        {
```

Comment:
make method private

Reply:
no this is called from the command line, can't be private

Reply:
then make it public